src/02-requirements/README.md

```diff
@@ -8,96 +8,31 @@ the fact that `extern crate` is not necessary in the 2018 edition.
-  because the board includes this functionality on-board. If you have an older
-  revision of the board then you'll need this module for chapters 10 and 11. For
-  completeness, we'll include instructions for using a Serial module. The book
-  will use [this particular model][sparkfun] but you can use any other model as
-  long as it operates at 3.3V. The CH340G module, which you can buy
- from [e-commerce][4] sites works too and it's probably cheaper for you to get.
-
-[sparkfun]: https://www.sparkfun.com/products/9873
-[4]: https://www.aliexpress.com/wholesale?SearchText=CH340G
-
-<p align="center">
-<img title="A 3.3v USB <-> Serial module" src="../assets/serial.jpg">
-</p>
+- A [micro:bit] board.
 
-- OPTIONAL. A HC-05 Bluetooth module (with headers!). A HC-06 would work too.
+[micro:bit]: https://microbit.org/
 
-(As with other Chinese parts, you pretty much can only find these on [e-commerce][5] [sites][6].
-(US) Electronics suppliers don't usually stock these for some reason)
+(You can purchase this board from several [electronics][0] [suppliers][1])
 
-[5]: http://www.ebay.com/sch/i.html?_nkw=hc-05
-[6]: https://www.aliexpress.com/wholesale?SearchText=hc-05
+[0]: https://microbit.org/buy/
+[1]: https://www.mouser.com/microbit/_/N-aez3t?P=1y8um0l
 
 <p align="center">
-<img title="The HC-05 Bluetooth module" src="../assets/bluetooth.jpg">
+<img title="micro:bit" src="../assets/microbit.jpg">
 </p>
 
-- Two mini-B USB cables. One is required to make the STM32F3DISCOVERY board work. The other is only
-  required if you have the Serial <-> USB module. Make sure that the cables both
-  support data transfer as some cables only support charging devices.
+- One micro-B USB cable. One is required to make the micro:bit board work.
+  Make sure that the cable supports data transfer as some cables only support charging devices.
 
 <p align="center">
 <img title="mini-B USB cable" src="../assets/usb-cable.jpg">
 </p>
 
-> **NOTE** These are **not** the USB cables that ship with pretty much every Android phone; those
-> are *micro* USB cables. Make sure you have the right thing!
-
-- MOSTLY OPTIONAL. 5 female to female, 4 male to female and 1 Male to Male *jumper* (AKA Dupont)
-  wires. You'll *very likely* need one female to female to get ITM working. The other wires are only
-  needed if you'll be using the USB <-> Serial and Bluetooth modules.
-
-(You can get these from electronics [suppliers][7] or from [e-commerce][8] [sites][9])
-
-[7]: https://www.adafruit.com/categories/306
-[8]: http://www.ebay.com/sch/i.html?_nkw=dupont+wire
-[9]: https://www.aliexpress.com/wholesale?SearchText=dupont+wire
-
-<p align="center">
-<img title="Jumper wires" src="../assets/jumper-wires.jpg">
-</p>
+> **NOTE** Some of the micro:bit kits already do ship with such cables. Furthermore lots of USB
+> cables used to charge your mobile are also micro-B ones, these cables *might* however be missing
+> the capability to transmit data.
```

Comment:
This wording feels a little weird to me -- perhaps something like this?

> **NOTE** You may already have a cable like this, as some of the micro:bit kits ship with such cables. Some USB Cables used to charge a mobile phone may also work, if they have the capability to transmit data.

Reply:
Your's sound a little easier to ...unfold? I guess... in my head, I"ll add it.